app/styles/components/utilities/_expandable-data-table.scss

```diff
@@ -0,0 +1,239 @@
+$rowSize: 1.6;
+$headerFontSize: 15px;
+$tableSize: ($rowSize * $rows) + $rows;
+$numOfColumns: 3;
+
+@mixin meta-icon-padding() {
+  @include rem('padding-right', 10px);
+  @include rem('padding-left', 2px);
+}
+
+.tooltip-inner {
+  max-width: none;
+  white-space: normal;
+}
+
+.expandable-data-table {
+
+  .toggle {
+    border: none;
+    display: inline-block;
+
+  }
+
+  .fa { color: #585858; }
+
+  ul.nav { display: inline-block; }
+
+  ul.nav li {
+    display: inline-block;
+    border: 1px solid #ddd;
+    border-right: none;
+
+    span { display: none; }
+  }
+
+  ul { }
+  ul li { }
+  ul li + li { @include rem('margin-top', $global-padding); }
+}
+
+
+
+.hidden { display: none; }
+.tabname { @include rem('padding-left', 5px); }
+.tab-content { @include rem('padding', 0); }
+.noAvailableData {
+  @include rem('margin-top', $global-padding);
+  @include rem('margin-left', 10px);
+  text-align: center;
+}
+
+
+.expandable-data-table .tab-pane {
+  padding: 0;
+
+  &.active {
+
+  }
+
+  //.table-responsive {
+  //  @include rem('margin-bottom', 10px);
+  //  border: none;
+  //
+  //  thead {
+  //    display: block;
+  //    width: 100%;
+  //  }
+  //
+  //  tr {
+  //    border: none;
+  //    display: block;
+  //    width: 100%;
+  //  }
+  //
+  //  tbody tr td {
+  //    white-space: nowrap;
+  //    display: inline-block;
+  //    width: 100% / $numOfColumns;
+  //  }
+  //
+  //  tbody {
+  //    display: block;
+  //    border-left: 1px solid #ddd;
+  //    border-bottom: 1px solid #ddd;
+  //    border-top: 1px solid #ddd;
+  //    border-right: none;
+  //  }
+  //
+  //  thead tr th {
+  //    @include font-size($headerFontSize);
+  //    white-space: nowrap;
+  //    display: inline-block;
+  //    width: 100% / $numOfColumns;
+  //    text-align: center;
+  //    background: $gray-light;
+  //    border-top: 1px solid #ddd;
+  //    border-bottom: 1px solid #ddd;
+  //  }
+  //
+  //  tbody tr td {
+  //    text-align: center;
+  //    font-weight: normal;
+  //    border-bottom: 1px solid #ddd;
+  //    border-top: none;
+  //    border-left: none;
+  //  }
+  //
+  //  table { border: none; }
+  //  thead tr { @include rem('padding-right', 16px); }
+  //  thead tr th:first-child { border-left: 1px solid #ddd; }
+  //  thead tr th:last-child { border-right: 1px solid #ddd; }
+  //  tbody tr:nth-last-child(1) { td { border-bottom: none; } }
+  //  tbody tr:nth-child(even) { background: $gray-light; }
+  //  tbody tr:nth-child(odd) { background: none; }
+  //  tbody tr td:nth-child(even) { background: none; }
+  //  tbody tr td:first-child { background: none; }
+  //}
+}
+
+.table-responsive.expanded {
+  tbody {
+    overflow-y: scroll;
+    max-height: none;
+  }
+}
+
+.table-responsive.scroll {
+  @include media-query(sm) {
+    thead tr {
+      @include rem('padding-right', 0);
+    }
+  }
+
+  tbody {
+    @include rem('max-height', $tableSize);
+    overflow-y: scroll;
+  }
+}
+
+//.multipleSingleColumnTables,
```

Comment:
Not sure why custom scaffolding would be necessary here, bootstrap scaffolding works fine